python/transformations/NY-Bikes-Predictions/requirements.txt

```diff
@@ -1,5 +1,5 @@
-quixstreaming==0.4.7a202206201538
+quixstreaming==0.4.9a202210131245
```

Comment:
@anandshah123 This change doesn't seem like it was intended. Can you check this please?